ant_world_db_creator/ant_world_db_creator.cc

```diff
@@ -233,13 +228,13 @@ int main(int argc, char *argv[])
             currentGridX++;
 
             // If we've reached the X edge of the world, move to start of next Y
-            if((worldMin[0] + ((float)currentGridX * gridSpacingM)) > worldMax[0]) {
+            if(worldMin[0] + currentGridX * gridSpacing > worldMax[0]) {
                 currentGridY++;
                 currentGridX = 0;
             }
 
             // If we've reached the Y edge of the world, stop
-            if((worldMin[1] + ((float)currentGridY * gridSpacingM)) > worldMax[1]) {
+            if(worldMin[1] + currentGridY * gridSpacing > worldMax[1]) {
```

Comment:
Why? I quite like making precedence explicit...

Reply:
It's more that I wanted to remove the cast to ``float``: I'll put the brackets back in.